mobile-app/app/components/CollateralFactorTag.tsx

```diff
@@ -1,21 +1,28 @@
+import { StyleProp, TextStyle, ViewStyle } from "react-native";
 import { tailwind } from "@tailwind";
 import BigNumber from "bignumber.js";
 import { ThemedTextV2, ThemedViewV2 } from "./themed";
 
 export function CollateralFactorTag({
   factor,
+  containerStyle = tailwind(
+    "h-5 flex flex-row items-center px-2 rounded border-0.5"
+  ),
+  textStyle = tailwind("text-xs font-semibold-v2"),
```

Comment:
```suggestion
  containerStyle = [
    tailwind("h-5 flex flex-row items-center px-2 border-0.5"),
    { borderRadius: 5 },
  ],
```

not really needed but if we follow exactly 100% on figma :v :v :v

Reply:
thanks, didn't catch that. let's keep it to 4px, so it'll be consistent with other collateral factor ui and less custom styling for us :D